.github/workflows/lint.yml

```diff
@@ -11,7 +11,11 @@ jobs:
       github.event_name == 'push' || github.event.pull_request.head.repo.full_name !=
       github.repository
 
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macOS-latest, windows-latest]
```

Comment:
It doesn't actually work on Windows: https://github.com/psf/black/actions/runs/6939494268/job/18876939284?pr=4060. I don't think we gain much from running this on multiple OSes so I'll just revert this part of the change.

Reply:
I thought I saw this pass ...

The benefit is the run self running on the different OS's and it also lets us know our linting has a good chance to run for developers on every OS ...